pkgs/top-level/all-packages.nix

```diff
@@ -6320,8 +6320,6 @@ with pkgs;
 
   ### DEVELOPMENT / TOOLS
 
-  activator = callPackage ../development/tools/activator { };
```

Comment:
I'm not sure if we have a general policy on this, but I generally appreciate it when instead of removing derivation attributes, we replace them with something like `throw "Typesafe Activator was removed in X for reason Y. See https://github.com/NixOS/nixpkgs/pull/25616 for more information"`, and then eventually we delete the attribute. This gives people a way to figure out what went wrong (although `nix-env` will just break silently, but fixing that is more work) even if they can't get the package.

I think there are probably better ways to do this in a more systematic way, but until we've agreed on those, I'd prefer a `throw`